assets/stylesheets/basic.css

```diff
@@ -176,4 +176,11 @@ input[type='submit'] {
 input[type='submit']:hover {
   background-color: rgba(255,255,255,.2);
   border: none;
-}
\ No newline at end of file
+}
+
+@media (max-width: 576px) {
```

Comment:
You need to put the other `.wrap` in a media query, too, or it will load both images on mobile.